src/interactions/views.py

```diff
@@ -99,17 +99,43 @@ def edit_comment(request, *, comment_id):
     )
 
 
+@require_http_methods(["GET"])
+def get_page_comments(request, *, pk):
+    comment_form_submission_url = reverse("interactions:comment-on-page", args=[pk])
+    page = get_object_or_404(Page, id=pk).specific
+    comments = []
+    for page_comment in comments_service.get_page_comments(page):
+        comment = comments_service.comment_to_dict(page_comment)
+        comments.append(comment)
```

Comment:
This could be simplified down to:

```suggestion
    comments = [
        comments_service.comment_to_dict(page_comment)
        for page_comment in comments_service.get_page_comments(page):
    ]
```